sound/soc/sof/sof-client-fw-gdb.c

```diff
@@ -22,36 +24,57 @@
 #include "ops.h"
 
 #define SOF_IPC_CLIENT_SUSPEND_DELAY_MS	3000
+#define SOF_FS_GDB_POLL_DELAY_MS 5
+
+union sof_ipc_gdb_message {
+	struct {
+		struct sof_ipc_cmd_hdr hdr;
+		char cmd[];
+	} ipc3;
+	struct {
+		struct sof_ipc4_msg msg;
+		char cmd[];
+	} ipc4;
+} __packed;
```

Comment:
looks like this can be squashed.

Reply:
sorry, I meant you are adding a new file to the kernel in a series of patches. You should just do it a 1 patch for the new file.

Reply:
@lgirdwood well, I tried to keep the first patch as close to the author's original as possible. But if we decide that that isn't needed, sure, can squash